lib/chef-dk/command/verify.rb

```diff
@@ -79,8 +79,14 @@ def components
         # We can't run it right now since graphviz specs are included in the
         # test suite by default. We will be able to switch to that command when/if
         # Graphviz is added to omnibus.
-        c.unit_test { sh("bundle exec rspec --color --format progress spec/unit --tag ~graphviz") }
-        c.integration_test { sh("bundle exec cucumber --color --format progress --tags ~@no_run --tags ~@spawn --tags ~@graphviz --strict") }
+        c.unit_test do
+          sh("bundle install")
```

Comment:
I guess having chef-acceptance makes me feel better about it. To clarify, I'm not concerned about things that `bundle check` would detect, I'm concerned that there can be bugs that go away when development dependencies are present (this has happened with chef-client before).